core/config/constants.js

```diff
@@ -76,12 +75,12 @@ const DESKTOP_EMULATION_METRICS = {
 };
 
 const screenEmulationMetrics = {
-  mobile: MOTOG4_EMULATION_METRICS,
+  mobile: MOTOGPOWER_EMULATION_METRICS,
   desktop: DESKTOP_EMULATION_METRICS,
 };
 
 
-const MOTOG4_USERAGENT = 'Mozilla/5.0 (Linux; Android 7.0; Moto G (4)) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/109.0.0.0 Mobile Safari/537.36'; // eslint-disable-line max-len
+const MOTOG4_USERAGENT = 'Mozilla/5.0 (Linux; Android 11; moto g power (2022)) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/109.0.0.0 Mobile Safari/537.36'; // eslint-disable-line max-len
```

Comment:
Should we capitalize `moto g power`?

Reply:
this is how it appeared on my device

Reply:
works for me

Reply:
fyi we went with `Mozilla/5.0 (Linux; Android 11; moto g power) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/%s Mobile Safari/537.36`